W 10673653 how to find org ID (#184)
mulesoft/docs-access-management: modules/ROOT/pages/organization.adoc

You can also view the organization ID, client ID, and client secret. These values apply to the root organization and grant permissions for all of the business groups contained within.

== Find your Organization ID

Some operations require you to specify your organization ID, or _orgId_. You also need your orgId to designate a business group or root organization when creating certain types of requests.

> You can obtain your orgId by extracting a your organization URL, executing an Anypoint CLI command, or using a token to invoke the Anypoint Platform REST API.

Reviewer: ```suggestion
You can obtain your orgId from your organization URL, executing an Anypoint CLI command, or using a token to invoke the Anypoint Platform REST API.
```
